docs/introduction.rst

```diff
@@ -1,142 +1,131 @@
-  This is because Flatpak applications and runtimes are contained to not interfere
-  with the system altogether.
-- **Rootless install**: elevated privileges are not required when installing a Flatpak
-  application or a runtime.
-- **Sandboxed applications**: one of Flatpak's main goals is to increase the security of desktop
-  systems by isolating applications from one another. This is achieved using sandboxing and means
-  that, by default, applications that are run with Flatpak have limited access to the host environment.
-
-Flatpak has some major advantages over other universal approaches to distributing
-applications on Linux:
+  distribution, including non-GNU distributions, systemd-free distributions,
+  distributions with a read-only operating system (OS), and various architectures without requiring
+  the developer to have access to the relevant hardware.
+- **Space for innovation**: Flatpak allows distribution maintainers to focus on
+  innovating their distribution without being burdened by packaging concerns.
+- **Stability**: breakages in Flatpak applications won't affect the system,
+  since they run in isolated environments.
+- **Rootless install**: Flatpak does not require elevated privileges to install
+  applications or runtimes.
+- **Sandboxed applications**: Flatpak increases security, one of its main goals, by
+  isolating applications from one another and limiting their access to the host environment.
+
+Flatpak also offers advantages over other universal approaches to Linux application distribution:
 
 - **Decentralized by design**: while Flatpak does provide a centralized service for distributing
-  applications, it also allows decentralized hosting and distribution, so that
-  application developers or downstreams can host their own applications and
-  application repositories.
-- **Desktop integration**: Flatpak also offers native integration for the main Linux desktops, so that
-  users can easily browse, install, run and use Flatpak applications through
-  their existing desktop environment and tools.
-- **Space efficiency**: Flatpak deduplicates libraries and other files used by multiple
-  applications to save megabytes or even gigabytes worth of storage depending on
-  the amount of applications installed.
-- **Delta updates**: only changed files are downloaded for updates.
+  applications, it also offers decentralized hosting and distribution, allowing developers or
+  downstreams to host their own applications and application repositories.
+- **Desktop integration**: Flatpak offers native integration with the main Linux desktops,
+  allowing users to easily browse, install, run, and manage Flatpak
+  applications through their existing desktop environment and tools.
+- **Space efficiency**: Flatpak saves storage by deduplicating libraries and
+  other files used by multiple applications.
+- **Delta updates**: only changed files are downloaded during updates.
```

Comment:
Join these two "Space efficiency and delta updates"

Reply:
Maybe add something to the delta updates then `only changed files are downloaded during updates, saving network bandwidth`.